apiary.apib

```diff
@@ -1424,6 +1424,7 @@ Update a calendar event.
         + description (string, optional, nullable)
         + starts_at: `2016-02-04T16:00:00+00:00` (string, optional)
         + ends_at: `2016-02-04T18:00:00+00:00` (string, optional)
+        + work_type_id: `b37e2bc7-dea0-4fda-88e9-c092fb65667d` (string, optional)
```

Comment:
Q: What is a work type exactly? Are there other types possible? Should this be an embedded object then?

Reply:
https://developer.teamleader.eu/#/reference/general/work-types/worktypes.list

A `WorkType` (previously known as `topic`) is what is now known in Teamleader as a "todo kind". Using the example of a web development agency, this might be "Development", "Design", "Operations", etc.

Reply:
Right :D sorry should've read the documentation 🤔